server/tests/src/test/java/org/infinispan/server/security/AuthenticationKeyCloakIT.java

```diff
@@ -34,7 +34,7 @@
 
 
    @ClassRule
-   public static KeyCloakServerRule KEYCLOAK = new KeyCloakServerRule("keycloak/infinispan-keycloak-realm.json");
+   public static KeyCloakServerRule KEYCLOAK = new KeyCloakServerRule("keycloak/infinispan-rhsso-realm.json","registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4-5");
```

Comment:
If you want to be able to specify alternate images, you should have a way to override using a system property. See the DB tests for an example.